.github/workflows/build-system-test-react-native.yml

```diff
@@ -162,7 +140,7 @@ jobs:
           PKG_MANAGER: ${{ matrix.pkg-manager }}
           DEPENDENCIES: ${{ env.DEPENDENCIES }}
```

Comment:
Related to comment above, can we remove the reliance on `DEPENDENCIES`.

Reply:
Done 88b528c59adc4e2248d3f56e4374ac9216c031cf